docs/source/user-guide/index.rst

```diff
@@ -2,10 +2,42 @@
 User guide
 **********
 
+Welcome to the conda build user guide. Here you can find
+tutorials and recipes as well as information about environment
+variables and wheel files.
+
 .. toctree::
```

Comment:
consider marking this as "hidden", so it shows in the TOC tree, but isn't duplicated here, as your intro blurbs serve as a nicer TOC.